source/reference/collection-level-filtering.txt

```diff
@@ -17,6 +17,62 @@ Filtered Sync
 
 .. include:: /includes/api/facts/filtering-intro.rst
 
+Starting in 1.6, ``mongosync`` supports both inclusion and exclusion filters 
+on sync, allowing you to fine tune what databases and collections on the source 
+cluster get synced to the destination.
+
+- With inclusion filters, ``mongosync`` syncs matching
+  databases and collections.
+- With exclusion filters, ``mongosync`` syncs all databases
+  and collections, except for those that match the filters.
+- With both inclusion and exclusion filters, ``mongosync`` only syncs
+  databases and collections that match the inclusion filter.   It ignores
+  the exclusion filters.
+- With no filters, ``mongosync`` syncs all databases and collections.
+
+.. _c2c-filter-syntax:
+
+Filter Syntax
+-------------
+
+The :ref:`c2c-api-start` API endpoint accepts two fields that configure 
+filtered sync: ``includeNamespaces`` and ``excludeNamespaces``.
+Each field takes an array of filters that specify the databases and collections 
+to include or exclude from sync.
+
+.. note::
+
+   If the :ref:`c2c-api-start` call uses both ``includeNamespaces`` and 
+   ``excludeNamespaces`` parameters, ``mongosync`` uses the inclusion 
+   filters and ignores the exclusion filters.
+
+Filters have the following syntax:
```

Comment:
[question] Would it be redundant to include syntax for both `includeNamespaces` and `excludeNamespaces`? Otherwise, maybe we should re-word the intro sentence to say something along the lines of: 

"For example, inclusion filters have the following syntax:"

Reply:
@ajhuh-mdb  Inclusion and exclusion filters have the same syntax.  I worry about listing them together since the user can't run them together.

Reply:
Fixed